apps/website/lib/llm/handler.ts

```diff
@@ -0,0 +1,117 @@
+import { NextRequest } from "next/server";
+import cors from "../../lib/cors";
+import {
+  CONTENT_TYPE_JSON,
+  CONTENT_TYPE_TEXT,
+  RequestBody,
+  LLMProviderConfig,
+} from "./types";
+
+export const runtime = "nodejs";
+export const preferredRegion = "auto";
+export const maxDuration = 300;
```

Comment:
are these needed here if they are on the route?

Reply:
These are not needed